### DISCUSSION TITLE
Add support of html/css to notification title

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#215] [General]
Currently, it is possible to use html/css in notification content and the tags will be rendered correctly in the browsers. 
It would be nice to have this feature for the title as well because currently HTML tags will be rendered as plain text.

**Comment by unamashana** [comment]
@RuslanHryn thanks for the suggestion. Where would this be useful?

**Reply by RuslanHryn** [reply]
@unamashana 
I see two use cases:
1. Styling the title for example changing the color etc.
2. Use as a workaround for localization of notification content and title. 
For example, we can send notifications that contain multiple languages and use CSS to hide/show content depending on the user's locale.

```
----- Notification title 
<div id="en">
Title in English 
</div>

<div id="de">
Content in German 
<div>
--------

CSS to show only English title.
#en { display: block; visibility: visible }
#de { display: none; visibility: hidden }
```

**Comment by unamashana** [comment]
@RuslanHryn in that case, I suggest using the JS SDKs to change styling, or build a custom inbox. Some examples at https://playground.magicbell.com/react
